src/main/java/net/jodah/failsafe/CircuitBreaker.java

```diff
@@ -15,34 +15,37 @@
  */
 package net.jodah.failsafe;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicInteger;
-import java.util.concurrent.atomic.AtomicReference;
-
 import net.jodah.failsafe.function.BiPredicate;
 import net.jodah.failsafe.function.CheckedRunnable;
 import net.jodah.failsafe.function.Predicate;
-import net.jodah.failsafe.internal.CircuitBreakerStats;
-import net.jodah.failsafe.internal.CircuitState;
-import net.jodah.failsafe.internal.ClosedState;
-import net.jodah.failsafe.internal.HalfOpenState;
-import net.jodah.failsafe.internal.OpenState;
+import net.jodah.failsafe.internal.*;
```

Comment:
Probably this should be made explicit?

Reply:
You mean the wildcard import? My IDE does this if there are several imports from the same package. The runtime cost should be nil.